docs/gke/iap.md

```diff
@@ -162,18 +162,74 @@ gcloud projects add-iam-policy-binding $PROJECT \
   --member user:${USER_EMAIL}
 ```
 
-### Self signed certificates and browser security warnings
+### Obtaining a valid certificate from LetsEncrypt
```

Comment:
Is there any requirement so that we can use this? Or anyone can just use it? Is it free?

Reply:
It's free - anyone can use it.